### PR TITLE
WIP serde_rustler 0.0.8 upgrade

### PR DESCRIPTION
**Pull request by lytedev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.3] - 2023-08-09
+### Changed
+- Upgrade Rustler for greater Erlant/OTP version compatibility
```
Comment (benhaney) [inline CHANGELOG.md]:
```suggestion
## [Unreleased]
### Changed
- Upgrade Rustler for greater Erlang/OTP version compatibility
```

File: native/jsonrs/Cargo.toml [hunk 2]
```diff
@@ -11,9 +11,9 @@ crate-type = ["dylib"]
 
 [dependencies]
 flate2 = "1.0"
-rustler = "0.28.0"
+rustler = "0.30.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-serde_rustler = { git = "https://github.com/benhaney/serde_rustler", tag = "v0.0.7" }
+serde_rustler = { git = "https://github.com/lytedev/serde_rustler", rev = "cd7c66fbc837135616a7ff39c13623de65e9f56f" }
```
Comment (benhaney) [inline native/jsonrs/Cargo.toml]: serde_rustler is tagged now
```suggestion
serde_rustler = { git = "https://github.com/benhaney/serde_rustler", tag = "v0.0.8" }
```
plus Cargo.lock change

**Review by benhaney — COMMENTED** [review]
LGTM with a tiny bit of cleanup. I'll do a release commit after

**Comment by benhaney** [comment]
Okay I guess I can just commit straight to your repo if you open a PR?

**Comment by lytedev** [comment]
Yeah there's a fancy checkbox that gives access if you want. 
image

**Pull request merged**